templates/index.html.jinja

```diff
@@ -9,7 +9,7 @@
 <body>
     <div id="container">
         <h1>Mastodon Digest</h1>
-        <span id="render-description">Rendered <span id="rendered-value">{{ rendered_at }}</span> from your timeline over the past <span id="hours-value">{{ hours }}</span> hours</span><br />
+        <span id="render-description">Rendered <span id="rendered-value">{{ rendered_at }}</span> from timeline “{{ timeline_name }}” over the past <span id="hours-value">{{ hours }}</span> hours</span><br />
```

Comment:
I've done a quick pass over this part of the template and group the config data into an expandable `<details>` element with a definition list to review the config. This will scale better when more options are added, while also tucking the debug info out of sight which makes the rendered page a little nicer to view.

There's probably more could be done for the template, but will keep this patch focused 😄